csharp/tests/test.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd )"
+
+rm -f csharp-tests
+rm -f libcsharp*.so
+dub build --arch=x86 --force
```

Comment:
Don't see how -r applies since it's files and not folders.

Reply:
I meant: "why are you deleting them?"

Reply:
To ensure that if there is compilation failure that the C# build will also fail. In automated testing that matters less since you're always starting with a clean dir, but in dev it was a time saver. Can remove if needed but it's not hurting anything.

Reply:
The way to do that is to make the script fail if any one the commands fail.

Otherwise to me it indicates a lack of faith in the build system if any build artifacts need to be removed (which is better done with `git clean -xfd` anyway).